hack/util.sh

```diff
@@ -707,9 +707,15 @@ function util::get_version() {
   git describe --tags --dirty
 }
 
+function util::get_revision() {
+  git rev-parse --short HEAD
+}
+
 function util::version_ldflags() {
   # Git information
   GIT_VERSION=$(util::get_version)
+  # Git revision
+  GIT_ABBREVIATIVE_COMMIT=$(util::get_revision)
```

Comment:
```suggestion
function util::version_ldflags() {
  # Git information
  GIT_VERSION=$(util::get_version)
  GIT_ABBREVIATIVE_COMMIT=$(git rev-parse --short HEAD)
```
How about inlining it?
PS: I can't tell much difference between `revision` and `version`.